LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Catppuccin
+Copyright (c) 2021 Catppuccin
```

Comment:
The port was created in 2022 so I think keeping the copyright year as 2022 makes sense? I've never really thought to downgrade the copyright years 🤔 
```suggestion
Copyright (c) 2022 Catppuccin
```

Reply:
I'm pretty sure we've been over this before somewhere in Discord. Nearly every single port uses the year 2021 in the LICENSE file sooooo...